charts/argo-services/templates/argo-workflows-rbac/service-account.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: argo-workflows-read-only
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    workflows.argoproj.io/rbac-rule: "'{{ .Values.rbacTeams.read_only }}' in groups"
+    workflows.argoproj.io/rbac-rule-precedence: "1"
```

Comment:
I wonder if the precedence should be swapped?

e.g. Person A is in gov-uk and gov-uk-production teams, they should be in read-write (admin) but now they will have a read-only role?

Reply:
Counter-intuitively, the higher the number the higher the precedence with argo workflows. So if the read-write rule has a higher number than the read-only, it will be selected over the read-only one.

Reply:
Thanks for that.